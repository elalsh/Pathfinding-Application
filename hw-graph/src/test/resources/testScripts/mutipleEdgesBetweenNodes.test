# Tests 2 nodes with multiple edges  but with different labels

# Create a graph
CreateGraph graph1

# Add nodes A
AddNode graph1 A
AddNode graph1 B

# Add an edges to the graph
AddEdge graph1 A B e1
AddEdge graph1 A B e2
AddEdge graph1 A B e3
AddEdge graph1 B A e4
AddEdge graph1 B A e5

# Print the nodes in the graph and the outgoing edges for src nodes to dest nodes
ListNodes graph1
ListChildren graph1 A
ListChildren graph1 B
