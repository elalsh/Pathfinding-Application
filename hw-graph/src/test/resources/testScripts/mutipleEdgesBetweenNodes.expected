# Tests 2 nodes with multiple edges  but with different labels

# Create a graph
created graph graph1

# Add nodes A
added node A to graph1
added node B to graph1

# Add an edges to the graph
added edge e1 from A to B in graph1
added edge e2 from A to B in graph1
added edge e3 from A to B in graph1
added edge e4 from B to A in graph1
added edge e5 from B to A in graph1

# Print the nodes in the graph and the outgoing edges for src nodes to dest nodes
graph1 contains: A B
the children of A in graph1 are: B(e1) B(e2) B(e3)
the children of B in graph1 are: A(e4) A(e5)
